modules/channel-messenger/src/config.ts

```diff
@@ -47,8 +47,17 @@ export interface Config {
    * @default 24h
    */
   chatUserAuthDuration: string
+
+  /**
+   * Allows to disable certain actions.
+   * Could be used to comply with some region's restrictions.
+   * Example: https://developers.facebook.com/blog/post/2020/12/04/upcoming-changes-messenger-api/
+   * @default []
+   */
+  blacklistedActions?: Array<MessengerAction>
```

Comment:
I think it make more sense to name this `disabledActions`, since it just means that Botpress is not sending that specific action, not that it is banned